nucypher/cli/actions/select.py

```diff
@@ -136,13 +136,23 @@ def select_client_account(emitter,
     return chosen_account
 
 
-def select_network(emitter: StdoutEmitter, message: Optional[str] = None) -> str:
+def select_network(emitter: StdoutEmitter, network_type: str, message: Optional[str] = None) -> str:
```

Comment:
👌 Nice reuse of the function for both networks.

Should we make the network type values ( "eth" / "polygon" ) constants? Callers of this function can use the contents as well.

Reply:
I'm happy to make that change, but not sure what the benefits are? it's just going to be:
```
ETH = "eth"
```
is that more readable?

Reply:
My original thought here was not just for the function itself, but that any callers of the function would then do:
```python
network = select_network(emitter=emitter, network_type=ETH)
```
OR
```python
network = select_network(emitter=emitter, network_type=POLYGON)
```

instead of having "eth" or "polygon" be hardcoded everywhere this function is called (~ 5 spots right now). Also a thought towards easily supporting other payment networks in the future. Presumably, the constants can live in `NetworksInventory`...?

wdyt?

Reply:
👍 if more networks end up being added this will be a cleaner solution

Reply:
> should really be "eth" and "payment"

In nuycpher-async I am calling them "identity" and "payment"